test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -87,6 +88,102 @@ public void CanAppendContent()
         }
 
         [Fact]
+        public void CanAppendFormatContent()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+
+            // Act
+            tagHelperContent.AppendFormat("{0} World!", "Hello");
+
+            // Assert
+            Assert.Equal("Hello World!", tagHelperContent.GetContent());
+        }
+
+        [Fact]
+        public void CanAppendFormat_WithAlignmentComponent()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+
+            // Act
+            tagHelperContent.AppendFormat("{0, -10} World!", "Hello");
+
+            // Assert
+            Assert.Equal("Hello      World!", tagHelperContent.GetContent());
+        }
+
+        [Fact]
+        public void CanAppendFormat_WithFormatStringComponent()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+
+            // Act
+            tagHelperContent.AppendFormat("0x{0:X}", 50);
+
+            // Assert
+            Assert.Equal("0x32", tagHelperContent.GetContent());
+        }
+
+        [Fact]
+        public void CanAppendFormat_WithCulture()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+
+            // Act
+            tagHelperContent.AppendFormat(CultureInfo.InvariantCulture, "{0} World!", "Hello");
```

Comment:
this test does not demonstrate use of the `IFormatProvider`.  use another `IFormatProvider` and an argument that gets formatted e.g. `new CultureInfo("fr-FR")` and `23.87`.